examples/basic/src/hello_world.cc

```diff
@@ -1,4 +1,4 @@
-#include "examples/basic/hello_world.h"
```

Comment:
I dont think this line is using absolute path, I think you are replacing one relative path with another. An absolute path would look something like this `C:code_base/examples/basic/hello_world.h` or `/home/code_base/examples/basic/hello_world.h`. The original line is actually the path from one of the "included" directories during the build process. 
I like the original way of declaring includes because it makes the project structure clear, for example imagine you were including interfaces from a parent module and you use `parent_module/includes/interfaces` instead of `../../includes/interfaces` it makes things clearer. However I don't think it matters a lot in this specific case, so I leave it up to you, this comment is not blocking.